metricbeat/modules.d/audit.yml.disabled

```diff
@@ -0,0 +1,3 @@
+- module: audit
+  metricsets: ["kernel"]
```

Comment:
I think you can remove it. The problem is that the same file is also used for the docs where it is probably nice to have it.